packages/monaco/src/browser/monaco-quick-input-service.ts

```diff
@@ -316,6 +317,15 @@ export class MonacoQuickInputService implements QuickInputService {
         const monacoPicks: Promise<QuickPickInput<IQuickPickItem>[]> = new Promise(async resolve => {
             const updatedPicks = (await picks).map(pick => {
                 if (pick.type !== 'separator') {
+                    const icon = pick.iconPath;
```

Comment:
So this code needs an update after the monaco upgrade? Can we mark it with a "todo" maybe?

Reply:
Good catch @tsmaeder! I added a todo comment